app/serializers/api/admin/commodities/commodity_csv_serializer.rb

```diff
@@ -0,0 +1,54 @@
+require_relative '../../../concerns/api/v2/csv/csv_serializer'
+
+module Api
+  module Admin
+    module Commodities
+      class CommodityCsvSerializer
+        include Api::V2::Csv::CsvSerializer
+
+        column :goods_nomenclature_sid, column_name: 'SID' do |row|
+          row[:goods_nomenclature_sid]
```

Comment:
I wonder if it would be better to return Commodity instances just to simplify this

Reply:
I think it would not work, in this case, we are serializing an array of hashes, returned from the fetch_chapter_commodities_for_date query.

Reply:
I don't think the data structure would be the problem. Possibly the attribute names between the commodity and the function result wouldn't match up totally, though.